configure_tls_protocol_version_and_ciphers.md

```diff
@@ -38,20 +30,27 @@ Although the SDK will default to the latest version of TLS by default, if one wi
 For example:
 
 ```csharp
-Microsoft.Azure.Devices.Shared.TlsVersions.Instance.SetTlsMinimumVersions(SslProtocols.Tls12);
+// For device and module clients - pass options into the client constructor.
+var options = new IotHubClientOptions
```

Comment:
Nick/Nate, I went ahead and updated this readme file because it'll be hard to detect as it won't be a build change.

Please note it for the migration doc.